File: C Programming (All Files)/C Programming (Lab Report 4)/Question_1.c
// Write a C Program To Define Some Constant Variable In Pre Processor.

#include <stdio.h>
#define MULTIPLY 25

int main()
{
    int num1;
    int result;

    printf("Please Enter a Number : ");
    scanf("%d", &num1);

    result = num1 * MULTIPLY;

    printf("The Given Number Multiplied By 25 Is : %d", result);

    return 0;
}

File: Python Programming (All Files)/Experiment 3 (Loops)/Question_1.py
# Write a Python Program To Find Factorial Of a Given Number.

n = int(input("Please Enter a Number To Find It's Factorial : "))

factorial = 1

for i in range(1, n + 1, 1):
    factorial = factorial * i

print("Factorial Of Given Number {} Is : {}".format(n, factorial))